bundle/src/main/java/com/jfrog/ide/eclipse/ui/ProjectsMap.java

```diff
@@ -0,0 +1,58 @@
+package com.jfrog.ide.eclipse.ui;
+
+import java.util.Comparator;
+import java.util.TreeMap;
+
+import org.apache.commons.lang3.StringUtils;
+import org.jfrog.build.extractor.scan.DependenciesTree;
+import org.jfrog.build.extractor.scan.GeneralInfo;
+
+/**
+ * Map of projects. Used in the UI to show the projects sorted by project name
+ * and path.
+ * 
+ * @author yahavi
+ */
+public class ProjectsMap extends TreeMap<ProjectsMap.ProjectKey, DependenciesTree> {
```

Comment:
Suggestion - move this class to the util package

Reply:
But it is not an utility class